README.md

```diff
@@ -106,7 +106,7 @@ For more examples, see [.markdownlint.jsonc][markdownlint-jsonc], [.markdownlint
 
 The CLI argument `--config` is not required.
 If it is not provided, `markdownlint-cli` looks for the file `.markdownlint.jsonc`/`.markdownlint.json`/`.markdownlint.yaml`/`.markdownlint.yml` in current folder, or for the file `.markdownlintrc` in the current or all parent folders.
-The algorithm is described in detail on the [`rc` package page][rc-standards].
+The algorithm is described in detail on the [`rc` package page][rc-standards]. Note that when relying on its lookup of `.markdownlintrc` in the current or parent folders, the file doesn't accept a `yaml` syntax (just INI and JSON), and it cannot have other extensions (such as `.json`), it must be named strictly `.markdownlintrc`.
```

Comment:
Let's please focus on what should be done rather than what should not. For example, maybe "the file format must be JSON or INI" and "the file must have that name" or maybe "the file must not have an extension".